vsphere/vmdk-to-ovf.sh

```diff
@@ -31,6 +31,12 @@ TEMPLATE_NAME=$6
 # Path to the vmdk to convert to a template
 VMDK_PATH=$7
 
+# Legacy bios or EFI
+LEGACY_BIOS=${8:--notset}
+LEGACYBIOS=0
+[ "$LEGACY_BIOS" = "-disableefi" ] && LEGACYBIOS=1
```

Comment:
The "usage" asks for "--disableefi", but we test for a single "-"

Reply:
Fixed.